### ISSUE TITLE
Aws sdk V2 in maintenance mode, upgrade for v3

### ISSUE THREAD

**Issue by colas-gosselin-wmx** [alexa/alexa-skills-kit-sdk-for-nodejs#729]
## I'm submitting a...
<pre><code>
[x ] Feature request
</code></pre>

## Feature request
The sdk still use aws-sdk v2, which is put to maintenance mode in 2023. 
It should be great to upgrade it to aws-sdk v3.

**Comment by allthepies** [comment]
I agree, error messages are now appearing in the lambda cloudwatch logs regarding AWS SDK v2 going into maintenance mode.  This is annoying as it is triggering cloudwatch alarms I have setup.

**Comment by aunt-kitty-codes** [comment]
This is happening for me, too. I'm in the middle of updating AWS SDK to v3 and am finding that the following happens - 

with this set up

```
const dynamoDBClient = new DynamoDB(dynamoDBClientConfig);

const longTermDataPersistenceAdapter: PersistenceAdapter = new DynamoDbPersistenceAdapter({
    dynamoDBClient,
    tableName: longTermTableName,
    partitionKeyName: 'userId',
});

```
we then get an error with the `dynamoDBClient` because the `DynamoDbPersistenceAdapter` relies on the `DynamoDB` class from AWS SDK v2.

**Comment by talkingnews** [comment]
> I agree, error messages are now appearing in the lambda cloudwatch logs regarding AWS SDK v2 going into maintenance mode. This is annoying as it is triggering cloudwatch alarms I have setup.

Crazy isn't it? Over 2 years now... but it looks like there's now a way of disabling these warning from clogging up your logs...

https://github.com/aws/aws-sdk-js/issues/4354

Either use env var `AWS_SDK_JS_SUPPRESS_MAINTENANCE_MODE_MESSAGE=1` 
or 
`require('aws-sdk/lib/maintenance_mode_message').suppress = true;`
